src/apps/media/src/app-revamp/components/UploadThumbnail.tsx

```diff
@@ -0,0 +1,33 @@
+import { Thumbnail } from "./Thumbnail";
+import { FC } from "react";
+import { Box } from "@mui/material";
+
+interface Props {
+  progress: number;
```

Comment:
Hmm, it can be optional as well. I'm just thinking that the UploadThumbnail should always contain the progress=100 so that it would completely display the thumbnail.